s3.go

```diff
@@ -147,7 +148,7 @@ func (s *S3Upload) sourceFiles() ([]*FileData, error) {
 		if _, err := io.Copy(h, file); err != nil {
 			return err
 		}
-		md5Hash := fmt.Sprintf("%x", h.Sum(nil))
+		md5Hash := base64.URLEncoding.EncodeToString(h.Sum(nil))
```

Comment:
I think this change breaks the check logic here: https://github.com/goware/s3up/pull/3/commits/de1f4b4e3d3a65d5331c68d4c7f417a5abd90b31#diff-5ce73d5a5749efd300763cc175977161R206
as ETag still contains 32 bytes md5